google/cloud/security/common/gcp_api/compute.py

```diff
@@ -14,11 +14,37 @@
 
 """Wrapper for Compute API client."""
 
+import httplib
+import socket
+
 from google.cloud.security.common.gcp_api._base_client import _BaseClient
 
+def is_compute_engine_instance():
```

Comment:
It seems a bit out of place to put this here. I think a module under ../util would be better.